lib/cubit/webhook/webhook_cubit.dart

```diff
@@ -1,153 +1,91 @@
-import 'dart:convert';
-
-import 'package:breez_preferences/breez_preferences.dart';
 import 'package:breez_sdk_liquid/breez_sdk_liquid.dart';
-import 'package:firebase_notifications_client/firebase_notifications_client.dart';
-import 'package:flutter/foundation.dart';
 import 'package:flutter_breez_liquid/flutter_breez_liquid.dart';
-import 'package:http/http.dart' as http;
 import 'package:hydrated_bloc/hydrated_bloc.dart';
 import 'package:l_breez/cubit/cubit.dart';
 import 'package:logging/logging.dart';
 
+export 'lnurl_pay_service.dart';
+export 'webhook_service.dart';
 export 'webhook_state.dart';
 
 final Logger _logger = Logger('WebhookCubit');
 
 class WebhookCubit extends Cubit<WebhookState> {
-  static const String notifierServiceURL = 'https://notifier.breez.technology';
-  static const String lnurlServiceURL = 'https://breez.fun';
-
   final BreezSDKLiquid _breezSdkLiquid;
-  final BreezPreferences _breezPreferences;
-  final NotificationsClient _notifications;
+  final WebhookService _webhookService;
+  final LnUrlPayService _lnUrlPayService;
 
-  WebhookCubit(
-    this._breezSdkLiquid,
-    this._breezPreferences,
-    this._notifications,
-  ) : super(WebhookState()) {
+  WebhookCubit(this._breezSdkLiquid, this._webhookService, this._lnUrlPayService) : super(WebhookState()) {
     _breezSdkLiquid.walletInfoStream.first.then(
       (GetInfoResponse getInfoResponse) => refreshWebhooks(walletInfo: getInfoResponse.walletInfo),
     );
   }
 
-  Future<void> refreshWebhooks({WalletInfo? walletInfo}) async {
-    _logger.info('Refreshing webhooks');
+  Future<void> refreshWebhooks({WalletInfo? walletInfo, String? username}) async {
```

Comment:
Is `username` needed here? I don't see a call to `refreshWebhooks()` with it

Reply:
Let's move our attention to new PR. It is now being utilized by the customize address bottom sheet.